src/diem/testing/suites/conftest.py

```diff
@@ -12,16 +12,19 @@
     dmw_stub_server,
     dmw_stub_diem_account_config,
     dmw_stub_diem_account_hrp,
+    dmw_stub_diem_id_domain,
 )
 from typing import Optional, Tuple, Dict, Any, Generator, Callable
 from dataclasses import asdict
 import pytest, json, uuid, requests, time, warnings
+import secrets
 
 
 @pytest.fixture(scope="package")
 def target_client(diem_client: jsonrpc.Client) -> RestClient:
+    domain = "target" + secrets.token_hex(8)
```

Comment:
I think instead of generating domain, should let cli set target-diem-id-domain similar with stub-diem-id-domain option

Reply:
Ignore my previous comment, I think we should not add domain for target app. The target app should be setup properly before run the test suite, which means user should add domain for the vasp as they need, thus here we should not assign one for the app, remove the domain here